package.json

```diff
@@ -37,6 +37,7 @@
     "eslint-plugin-promise": "^1.3.1",
     "eslint-plugin-standard": "^1.3.2",
     "five-bells-integration-test-loader": "^1.0.0",
+    "five-bells-shared": "~21.0.1",
```

Comment:
In the vein of https://github.com/interledgerjs/ilp-plugin-bells/issues/80, it would be good if we could separate the schemas into a separate repo (https://github.com/interledgerjs/five-bells-shared/issues/173) and only include those and the schema validation library in here.